dispatcher/backend/src/db/engine/__init__.py

```diff
@@ -0,0 +1,11 @@
+import os
```

Comment:
Nope, it was done so that Alembic could import the models without depending on this initialisation code which requires that you set the `POSTGRES_URI` env variable.
Now that I changed my mind on this and linked Alembic connectable in `env.py` to the same variable, it makes no sense to use a folder.